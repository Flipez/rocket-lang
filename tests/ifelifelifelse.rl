i = 0
while (i<4)
  if (i==0)
    puts("if/elif/el: if block")
  elif (i==1)
    puts("if/elif/elif/el: first elif block")
  elif (i==2)
    puts("if/elif/elif/el: second elif block")
  else
    puts("if/elif/else: else block")
  end

  i = i+1
end
