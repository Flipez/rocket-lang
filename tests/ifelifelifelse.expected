"if/elif/el: if block"
"if/elif/elif/el: first elif block"
"if/elif/elif/el: second elif block"
"if/elif/else: else block"
nil
